src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/service/handler/WarningTaskWorkerHandler.java

```diff
@@ -79,16 +79,24 @@ public void completeWarningTaskService(ExternalTask externalTask, ExternalTaskSe
 
     private void addWarningToDelayedProcesses(String caseId, String updatedWarningValues) {
         List<CamundaProcess> processes = getProcesses(caseId);
-        processes.forEach(process -> updateDelayedProcessWarnings(process, updatedWarningValues));
+        processes.forEach(process -> updateDelayedProcessWarnings(caseId, process, updatedWarningValues));
     }
 
-    private void updateDelayedProcessWarnings(CamundaProcess process, String warningToAdd) {
+    private void updateDelayedProcessWarnings(String caseId, CamundaProcess process, String warningToAdd) {
+        //todo: add some logs and hande null pointer
         String serviceToken = authTokenGenerator.generate();
         CamundaProcessVariables processVariables = camundaClient.getProcessInstanceVariables(
             serviceToken,
             process.getId()
         );
 
+        if (processVariables == null || processVariables.getProcessVariablesMap() == null) {
+            log.info("addWarningToDelayedProcesses processVariables not found. "
+                     + "caseId:{} warningToAdd:{} tenantId:{} processId:{}",
+                caseId, warningToAdd, process.getTenantId(), process.getId());
+            return;
```

Comment:
Should be a warning level log message

Reply:
Is the processId, caseId and warningsToAdd already output earlier in this process?
Otherwise there is a case for outputting a lot like that every time for debugging purposes.